downloadableTable selection of rows

ginberg opened the pull request:
-Add ability to set selected rows in downloadableTable

cb4ds commented on R/downloadableTable.R:
```diff
@@ -118,7 +120,7 @@ downloadableTableUI <- function(id,
 #'
 #' \strong{\code{callModule(downloadableTable, id, logger, filenameroot,
-#' downloaddatafxns, tabledata, rownames, caption)}}
+#' downloaddatafxns, tabledata, selection, rownames, caption)}}
```
need to move this to the end of the function for backwards compatibility for users who do not use named inputs.

cb4ds commented:
Is it possible to also add a test?  Thanks!

ginberg commented:
ok, I have moved the selection so that it's the last argument. What do you think of a manual test where we start the module in a shiny app and the tester has to check if the selection is set? See the test file that I added. I don't know how we can test it otherwise.
